lib/snowpack/odbc.ex

```diff
@@ -151,7 +151,7 @@ defmodule Snowpack.ODBC do
         _from,
         %{pid: pid} = state
       ) do
-    case :odbc.param_query(pid, to_charlist(statement), params, timeout) do
+    case :odbc.param_query(pid, :binary.bin_to_list(statement), params, timeout) do
```

Comment:
Why use `:binary.bin_to_list` instead of `to_charlist`?

The statement is expected to be a list of ASCII characters.
https://www.erlang.org/doc/man/odbc.html#param_query-3

Reply:
running `SELECT 'これはテスト文字列です'` was causing `to_charlist` to throw an exception. changing the handler to this caused it to be able to be sent to Snowflake

Reply:
Nice.

Reply:
Would it be appropriate to add a regression test to keep that bug from slipping back in in the future?

Reply:
there is an issue with getting those characters back out. I believe that ODBC is where they get munched, which is why we can't have a test for this :'( 

i could test that the query and params get passed to ODBC correctly but not round trip like the rest of the tests